products/neo-savant/BUILD

```diff
@@ -0,0 +1,266 @@
+    "//:node_modules/leader-line-types",
+    "//:node_modules/moment",
+    "//:node_modules/numbro",
+    "//:node_modules/rc-switch",
+    "//:node_modules/react",
+    "//:node_modules/react-ace",
+    "//:node_modules/react-apollo",
+    "//:node_modules/react-beautiful-dnd",
+    "//:node_modules/react-bootstrap",
+    "//:node_modules/react-contenteditable",
+    "//:node_modules/react-custom-scroll",
+    "//:node_modules/react-dom",
+    "//:node_modules/react-dropzone",
+    "//:node_modules/react-fontawesome",
+    "//:node_modules/react-router-dom",
+    "//:node_modules/react-table",
+    "//:node_modules/sanitize-html",
+    "//:node_modules/@types/enzyme",
+    "//:node_modules/@types/jest",
+    "//:node_modules/@types/node",
+    "//:node_modules/@types/react",
+    "//:node_modules/@types/react-beautiful-dnd",
+    "//:node_modules/@types/react-custom-scroll",
+    "//:node_modules/@types/react-dom",
+    "//:node_modules/@types/react-router-dom",
+    "//:node_modules/@types/react-table",
+    "//:node_modules/@types/react-test-renderer",
+    "//:node_modules/@typescript-eslint/eslint-plugin",
+    "//:node_modules/@typescript-eslint/parser",
+    "//:node_modules/cz-conventional-changelog",
+    "//:node_modules/enzyme",
+    "//:node_modules/enzyme-to-json",
+    "//:node_modules/eslint-plugin-react",
+    "//:node_modules/react-test-renderer",
+    "//:node_modules/process",
+    "//:node_modules/url",
+    "//:node_modules/assert",
+    "//:node_modules/crypto-browserify",
+    "//:node_modules/stream-http",
+    "//:node_modules/https-browserify",
+    "//:node_modules/os-browserify",
+    "//:node_modules/buffer",
+    "//:node_modules/stream-browserify",
+    "//:node_modules/string_decoder",
+    "//:node_modules/events",
+    "//:node_modules/classnames",
+    "//:node_modules/tslib",
+    "//:node_modules/bip39",
+    "//:node_modules/hash.js",
+    "//:node_modules/hdkey",
+]
+
+# cra_bin.react_app_rewired(
+#     # Note: If you want to change the name make sure you update BUILD_PATH below accordingly
+#     # https://create-react-app.dev/docs/advanced-configuration/
+#     name = "build",
+#     srcs = _RUNTIME_DEPS + [
+#         "//:node_modules/react-app-rewired",
+#         "//:node_modules/react-scripts",
+#         "//:node_modules/typescript",
+#         "//:node_modules/web-vitals",
+#         "//:node_modules/webpack",
+#     ],
+#     args = ["build"],
+#     chdir = package_name(),
+#     env = {
+#         "BUILD_PATH": "./build",
+#     },
+#     out_dirs = ["build"],
+# )
+
+# copy_to_bin(
+#     name = "copy_test_files",
+#     srcs = glob(_TESTS),
+# )
+
+# pkg_tar(
+#     name = "html-folder",
+#     srcs = [":build"],
+#     mode = "0755",
+#     package_dir = "/usr/share/nginx/html/",
+#     strip_prefix = "build",
+# )
+
+# pkg_tar(
+#     name = "nignx-conf",
+#     srcs = ["nginx/default.conf"],
+#     mode = "0755",
+#     package_dir = "/etc/nginx/conf.d/",
+#     deps = [],
+# )
+
```

Comment:
There are lots of comments here - are you sure you want to include them in the repo?